src/tools/crossgen/crossgen.cpp

```diff
@@ -544,6 +546,17 @@ int _cdecl wmain(int argc, __in_ecount(argc) WCHAR **argv)
             fLargeVersionBubbleSwitch = true;
         }
 #endif
+        else if (MatchParameter(*argv, W("BaseAddress")))
```

Comment:
And this place (the rest of the plumbing can stay)

Reply:
Done